### ISSUE TITLE
Remove arXiv version number from “file” field (with auto file download disabled)

### ISSUE THREAD

**Issue by BenjaminDAnjou** [JabRef/jabref#9770]
When using the arXiv fetcher (with auto file download disabled), the version number still appears in the “file” field. For instance, the “file” field after fetching without file download is:

file = {:http://arxiv.org/pdf/1701.02434v1:PDF},

As a result, the user is not automatically linked to the latest version. To be linked to the latest version everywhere, the link should be

file = {:http://arxiv.org/pdf/1701.02434:PDF},

Would it be possible to remove the version number in the “file” field so that we are automatically linked to the latest version?

Right now, the only way is to manually delete the version number in the “file” field, which is time consuming.

This was tested with JabRef 5.9 on MacOS Ventura 13.2.1.

**Comment by Siedlerchr** [comment]
Hm, if I remember correctly this should aleady be the case. 
Did you use New entry -> By id or did you use the web search?

**Comment by Siedlerchr** [comment]
@koppor Nope, it's totally valid to store urls in the file field. We have checks for this e.g. isOnlineLink

**Comment by koppor** [comment]
> @koppor Nope, it's totally valid to store urls in the file field. We have checks for this e.g. isOnlineLink

There should be a cleanup to move this to the url field. Otherwise, this is really strange IMHO.

My main point is that we have auto download in place. Hence, an online link there is an indicator of an error IMHO.

**Comment by koppor** [comment]
Source: https://docs.jabref.org/finding-sorting-and-cleaning-entries/filelinks

![Screenshot_2023-04-18-13-37-29-88_ffb2f5e1b976ff98cfc94f359fbce8de.jpg](https://user-images.githubusercontent.com/1366654/232765614-3192efd6-9bc8-4832-95cf-1aad51c2cd01.jpg)

**Comment by Siedlerchr** [comment]
This is perfectly fine. JabRef can handle http links very well in the LinkedFiles and you can do a right click -> Download

When auto download is not activated (as is here the case) the http link is stored in the file field

<img width="607" alt="grafik" src="https://user-images.githubusercontent.com/320228/232844831-fe95b70c-985c-4421-9e12-399dc3acaa29.png">
